nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -16,6 +16,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <AllowUnsafeBlocks Condition="'$(CsWinRTComponent)' == 'true'">true</AllowUnsafeBlocks>
     <AllowUnsafeBlocks Condition="$(CsWinRTEnabled)">true</AllowUnsafeBlocks>
     <CoreCompileDependsOn>CsWinRTIncludeProjection;CsWinRTRemoveWinMDReferences;$(CoreCompileDependsOn)</CoreCompileDependsOn>
+    <TrackFileAccess>false</TrackFileAccess> 
```

Comment:
I wonder if we should do a condition on if not CsWinRTComponent as authoring may and probably not be broken for incremental builds.

Reply:
I can do that, but why do you think authoring would not be affected?  It has to run cswinrt.exe as well, potentially with inputs from project references.

Reply:
I was thinking authoring might not be affected because of 2 reasons.
- If the winmd of the component changed, then the generated C# files generated by the CsWinRT tool that were passed as generated files to the source generator would have been seen as changed as the source generator is what writes the winmd too.
- If a dependent winmd from a project reference changed, it would mean the C#/WinRT projection of that winmd referenced by the authored component would also have been updated and it would be seen as a change.

Reply:
OK, added the condition - we can always tighten later if need by